swarm/storage/localstore/localstore.go

```diff
@@ -44,7 +44,7 @@ var (
 
 var (
 	// Default value for Capacity DB option.
-	defaultCapacity uint64 = 5000000
+	defaultCapacity uint64 = 30000000
```

Comment:
revert back to default `5000000`. this implicates all swarm users

Reply:
isn't our intention is to support 100gb for all users? Today we support only 20gb.